src/templates/plugin.jsx

```diff
@@ -73,14 +73,13 @@ function PluginPage({data: {jenkinsPlugin: plugin, reverseDependencies: reverseD
                         {state.selectedTab === 'documentation' && (<>
                             {plugin.wiki.content && <div className="content" dangerouslySetInnerHTML={{__html: plugin.wiki.content}} />}
                         </>)}
-                        {state.selectedTab === 'releases' && <PluginReleases pluginId={plugin.id} />}
+                        {state.selectedTab === 'releases' && <PluginReleases pluginId={plugin.id} versions={versions.edges.map(edge => edge.node)} />}
                         {state.selectedTab === 'issues' && <PluginIssues pluginId={plugin.id} />}
                         {state.selectedTab === 'dependencies' && <PluginDependencies dependencies={plugin.dependencies} reverseDependencies={reverseDependencies.edges}/>}
                     </div>
                 </div>
                 <div className="col-md-3 gutter">
-                    <a href={`https://updates.jenkins.io/download/plugins/${plugin.name}`}
```

Comment:
A second redundant link to the same tab is redundant.

Finding this out during advisory publication is 👎